.gitignore

```diff
@@ -9,3 +9,6 @@
 /profile.out
 /coverage.txt
 release/
+
+# Docusaurus
+/docs/docusaurus/
```

Comment:
How did you manage to create this directory? or when does this gets created?
I'm able to run the site without this directory. 🤔

Reply:
Doh! I have no idea where that came from and assumed it was from building docusaurus. I can't repro either, removing that commit. now. 😊 Thanks for checking!